lisa/operating_system.py

```diff
@@ -383,15 +383,20 @@ def capture_system_information(self, saved_path: Path) -> None:
         )
         from lisa.tools import Chmod, Find
 
-        find_tool = self._node.tools[Find]
-        file_list = find_tool.find_files(
-            self._node.get_pure_path("/var/log/azure/"),
-            file_type="f",
-            sudo=True,
-            ignore_not_exist=True,
-        )
-        if len(file_list) > 0:
-            self._node.tools[Chmod].update_folder("/var/log/azure/", "a+rwX", sudo=True)
+        file_list = []
```

Comment:
@LiliDeng is working on this, and mostly done.

https://github.com/microsoft/lisa/pull/2517

Reply:
#2517 was merged. please rebase to the latest code.